### PR TITLE
Implemented create button

### PR DESCRIPTION
**Pull request by jyao244**
# Description

Implemented create button.

# Screenshots
捕获
捕获

## Type of change

Please delete options that are not relevant.

- [x] **New feature** (non-breaking change which adds functionality)

# Checklist:

Leave blank if not applicable

I have completed these steps when making this pull request:

- [x] I have assigned my name to the issue
- [x] I have moved the issue to the **In Progress** column
- [x] I have labelled the PR appropriately
- [x] I have assigned myself to the PR

Before opening the PR for review:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added attributions to new dependencies and resources
- [x] I have moved the linked issue to the **Review in Progress** column

### CODE REVIEW

File: packages/frontend/src/components/notes/note/newNoteModal.tsx [hunk 1]
```diff
@@ -38,6 +40,37 @@ const NewNoteModal: React.FC<NewNoteModalProps> = ({
...
+
+  // const { data, mutate } = useApi('/api/v1/house/note', { method: 'get' });
+
+  const createNote = useApiMutation('/api/v1/house/note', { method: 'post' });
+
+  async function onClickCreate() {
+    try {
+      const { name, value, type } = newNoteDetails;
+      await createNote({ body: { name, value, type } });
+      // mutate();
```
Comment (cdra334) [inline packages/frontend/src/components/notes/note/newNoteModal.tsx:69]: Does mutate() not need to be included?

**Review by cdra334 — APPROVED** [review]
LGTM! I liked the use of the enum here.

File: packages/frontend/src/components/notes/note/newNoteModal.tsx (hunk 1 above)
Comment (noahjade) [reply packages/frontend/src/components/notes/note/newNoteModal.tsx:69]: Also curious about this!

**Review by noahjade — APPROVED** [review]
Awesome work! The enum is really cool and keeps the code clean ✨ I'm also curious about the mutate(), but if it works it works haha

**Pull request merged**